packages/insured-bridge-relayer/src/RelayerConfig.ts

```diff
@@ -88,8 +90,9 @@ export class RelayerConfig {
     this.botModes = {
       relayerEnabled: RELAYER_ENABLED === "true" ? true : false,
       disputerEnabled: DISPUTER_ENABLED === "true" ? true : false,
-      finalizerEnabled: FINALIZER_ENABLED === "true" ? true : false,
```

Comment:
NOTE BREAKING CHANGE!

I renamed these variables and envs to be more descriptive, given we now have a new module.